lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -155,6 +171,15 @@ public APIGatewayProxyResponseEvent handleRequest(
                             govukSigninJourneyId,
                             ipAddress);
 
+            if (configService.enabled(CoreFeatureFlag.INHERITED_IDENTITY)) {
```

Comment:
Here's the important stuff. Behind a feature flag it tries to validate and story any inherited identities